integrationTests/vm/arwen/utils.go

```diff
@@ -20,6 +23,14 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// TODO: Rename this file to "utils_test.go".
```

Comment:
Don't suffix a file name with `_test` unless it contains test functions. Otherwise, `go test` will search for test functions in it.

Reply:
Removed TODO.